public/app/features/panel/panellinks/link_srv.ts

```diff
@@ -235,22 +235,26 @@ export interface LinkService {
   getDataLinkUIModel: <T>(link: DataLink, scopedVars: ScopedVars, origin: T) => LinkModel<T>;
 }
 
+export interface LinkServiceDependencies {
+  templateSrv: TemplateSrv;
+  getTimeSrv: () => TimeSrv;
```

Comment:
feels nicer to have this be just timeSrv: TimeSrv 
and have the default value call getTimeSrv

Reply:
yeah I like that suggestion!